guides/common/modules/proc_configuring-group-mapping-for-keycloak-authentication.adoc

```diff
@@ -1,20 +1,20 @@
 [id="configuring-group-mapping-for-keycloak-authentication_{context}"]
-= Configuring group mapping for {keycloak} authentication
+= Optional: Configuring external group mapping for {keycloak} authentication
 
-Optionally, to implement the Role Based Access Control (RBAC), create a group in {Project}, assign a role to this group, and then map an Active Directory group to the {Project} group.
-As a result, anyone in the given group in {keycloak} are logged in under the corresponding {Project} group.
-This example configures users of the {Project}-admin user group in the Active Directory to authenticate as users with administrator privileges on {Project}.
+To implement the Role Based Access Control (RBAC), create a group in {Project}, assign a role to this group, and then map an Active Directory group to the {Project} group.
+As a result, anyone in the given group in {keycloak} will log in under the corresponding {Project} group.
+
+For example, you can configure users of the {Project}-admin user group defined in Active Directory to authenticate as users with administrator privileges on {Project}.
```

Comment:
```suggestion
For example, you can configure users of the {Project}-admin user group defined in Active Directory to authenticate as users with administrator privileges on {Project}.

If a group mapping is not configured every user will receive the _Default role_ permissions.
```

There was a Forum thread recently, where this was a key point, might be good to add 🙂 
I'm not sure right now if the permissions will get reset on every login, if you manually add a role in Foreman, but I kinda expect that to happen.

Reply:
I assume it's this one: https://community.theforeman.org/t/default-role-assigned-to-new-user-rh-sso-external-way/38634/2  Adding just for reference.